app/code/Magento/LayeredNavigation/Test/Unit/Observer/Edit/Tab/Front/ProductAttributeFormBuildFrontTabObserverTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\LayeredNavigation\Test\Unit\Observer\Edit\Tab\Front;
+
+use Magento\Config\Model\Config\Source\Yesno;
+use Magento\Framework\Data\Form;
+use Magento\Framework\Data\Form\Element\Fieldset;
+use Magento\Framework\Event\Observer;
+use Magento\Framework\Module\Manager;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\LayeredNavigation\Observer\Edit\Tab\Front\ProductAttributeFormBuildFrontTabObserver;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Unit Test for \Magento\LayeredNavigation\Observer\Edit\Tab\Front\ProductAttributeFormBuildFrontTabObserver
+ */
+class ProductAttributeFormBuildFrontTabObserverTest extends TestCase
+{
+    /**
+     * @var MockObject|Observer
+     */
+    private $eventObserverMock;
+
+    /**
+     * @var MockObject|Yesno
+     */
+    private $optionListLock;
+
+    /**
+     * @var MockObject|Manager
+     */
+    private $moduleManagerMock;
+
+    /**
+     * @var ProductAttributeFormBuildFrontTabObserver
+     */
+    private $observer;
+
+    /**
+     * @inheritDoc
+     */
+    protected function setUp()
+    {
+        $this->optionListLock = $this->createMock(Yesno::class);
+        $this->moduleManagerMock = $this->createMock(Manager::class);
+        $this->eventObserverMock = $this->getMockBuilder(Observer::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['getForm'])
+            ->getMock();
+
+        $objectManager = new ObjectManager($this);
+        $this->observer = $objectManager->getObject(
+            ProductAttributeFormBuildFrontTabObserver::class,
+            [
+                'optionList' => $this->optionListLock,
+                'moduleManager' => $this->moduleManagerMock,
+            ]
+        );
+    }
+
+    /**
+     * Test case when module output is disabled
+     */
+    public function testExecuteWhenOutputDisabled()
```

Comment:
It's not critical but would be preferred to follow the strict type of definition for methods signature.
```suggestion
    public function testExecuteWhenOutputDisabled(): void
```

Reply:
done